CHANGELOG.md

```diff
@@ -43,6 +43,7 @@ Released on 2021-01-18
 
 ### Rule Changes
 
+* rule(Sudo Potential Privilege Escalation (CVE-2021-3156)): new rule created to detect CVE-2021-3156[[#1543](https://github.com/falcosecurity/falco/pull/1543)] - [@darryk5](https://github.com/darryk5)
```

Comment:
Changelog must not be changed manually.

It gets automatically compiled from the `release-notes` block in the PR template.